evm/src/cpu/columns.rs

```diff
@@ -1,207 +1,204 @@
 // TODO: remove when possible.
 #![allow(dead_code)]
 
-use std::ops::Range;
+use std::borrow::{Borrow, BorrowMut};
+use std::mem::{size_of, transmute, transmute_copy, ManuallyDrop};
+use std::ops::{Deref, DerefMut};
 
 use crate::memory;
 
-/// Filter. 1 if the row is part of bootstrapping the kernel code, 0 otherwise.
-pub const IS_BOOTSTRAP_KERNEL: usize = 0;
-
-/// Filter. 1 if the row is part of bootstrapping a contract's code, 0 otherwise.
-pub const IS_BOOTSTRAP_CONTRACT: usize = IS_BOOTSTRAP_KERNEL + 1;
-
-/// Filter. 1 if the row corresponds to a cycle of execution and 0 otherwise.
-/// Lets us re-use decode columns in non-cycle rows.
-pub const IS_CPU_CYCLE: usize = IS_BOOTSTRAP_CONTRACT + 1;
-
-/// If CPU cycle: The opcode being decoded, in {0, ..., 255}.
-pub const OPCODE: usize = IS_CPU_CYCLE + 1;
-
-// If CPU cycle: flags for EVM instructions. PUSHn, DUPn, and SWAPn only get one flag each. Invalid
-// opcodes are split between a number of flags for practical reasons. Exactly one of these flags
-// must be 1.
```

Comment:
NB: I meant to bring back these comments before making a PR. I’ll copy them over